src/ert/shared/cli/main.py

```diff
@@ -48,6 +49,7 @@ def run_cli(args):
         return
 
     evaluator_server_config = EvaluatorServerConfig(custom_port_range=args.port_range)
+    exp_id = str(uuid.uuid1()).split("-", maxsplit=1)[0]
```

Comment:
I don't really like this trimming of uuid, we should reconsider how to assign uuids/what to use (see also https://stackoverflow.com/a/56398787)

Reply:
Made it the full `uuid`